packages/rulesets/generated/spectral/az-common.js

```diff
@@ -613,6 +613,8 @@ const ruleset = {
             description: "Operation should have a summary or description.",
             message: "Operation should have a summary or description.",
             severity: "warn",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
```suggestion
            disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-core/documentation-required' rule.",
```

I believe this is the same rule